packages/shared/src/components/profile/ProfileButton.tsx

```diff
@@ -35,26 +38,40 @@ export default function ProfileButton({
           icon={<SettingsIcon />}
         />
       ) : (
-        <SimpleTooltip placement="left" content="Profile settings">
-          <button
-            type="button"
-            className={classNames(
-              'focus-outline h-10 cursor-pointer items-center gap-2 rounded-12 border-none bg-background-subtle p-0 font-bold text-text-primary no-underline typo-callout',
-              className ?? 'flex',
-            )}
-            onClick={wrapHandler(() => onUpdate(!isOpen))}
-          >
-            <ReputationUserBadge
-              className="ml-3 !typo-callout"
-              user={user}
-              iconProps={{
-                size: IconSize.Medium,
-              }}
-              disableTooltip
+        <div className="flex h-10 items-center rounded-12 bg-surface-float px-1">
+          {isStreaksEnabled && (
+            <ReadingStreakButton
```

Comment:
Why is a streaks button in the profile button?
It feels a bit like abuse?

Reply:
Ah right, sorry missed the context of what it looked like.
Since it's fully part of it now it seems ok maybe?

Reply:
It's ok in a way, it's better if it has another named wrapper - a better name for which is not really coming in my mind ATM.
Unless u're thinking of something, I think it's ok to move on like this as well